sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -1,5 +1,73 @@
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_slots")]
+        public Azure.AI.Personalizer.DecisionContextDocument[] Slots { get { throw null; } set { } }
+    }
+    public partial class DecisionContextDocument
+    {
+        public DecisionContextDocument(string id, System.Collections.Generic.List<string> json, string slotId, System.Collections.Generic.List<string> slotJson) { }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("f")]
+        public System.Collections.Generic.Dictionary<string, float[]> FloatFeatures { get { throw null; } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_tag")]
+        public string ID { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("j")]
+        public System.Collections.Generic.List<string> JSON { get { throw null; } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("i")]
+        public Azure.AI.Personalizer.DecisionContextDocumentId Marginal { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_id")]
+        public string SlotId { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("sj")]
+        public System.Collections.Generic.List<string> SlotJson { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("s")]
+        public Azure.AI.Personalizer.DecisionContextDocumentSource Source { get { throw null; } set { } }
+    }
+    public partial class DecisionContextDocumentId
+    {
+        public DecisionContextDocumentId() { }
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("constant")]
+        public int Constant { get { throw null; } set { } }
```

Comment:
What does this property `Constant` represent? Can we make it something more descriptive.

Reply:
This class is not necessary so I removed it.